package.json

```
@@ -178,7 +178,7 @@
 		"lint:jsdoc:fix": "wp-scripts lint-js --fix",
 		"env:start": "node ./tools/local-env/scripts/start.js",
 		"env:stop": "node ./tools/local-env/scripts/docker.js down",
-		"env:restart": "npm run env:stop && npm run env:start",
+		"env:restart": "npm run env:stop &s& npm run env:start",
```

It looks like accidental `s` inserted when trying to save the file:
```suggestion
		"env:restart": "npm run env:stop && npm run env:start",
```